lib/generators/sufia/cached_stats_generator.rb

```diff
@@ -16,7 +16,8 @@ def banner
   def copy_migrations
     [
       'create_file_view_stats.rb',
-      'create_file_download_stats.rb'
+      'create_file_download_stats.rb',
+      'create_work_view_stats.rb'
```

Comment:
We'll want this in a new generator (`cached_work_stats_generator.rb`?) so that folks who already have the other two migrations can pick the new one up independent of the old ones.